src/main/java/org/mongojack/internal/stream/JacksonDBObject.java

```diff
@@ -26,39 +24,30 @@
  */
 public class JacksonDBObject<T> extends BasicDBObject {
 
-    private T object;
+  private T object;
 
-    private Class<?> view;
+  private Class<?> view;
 
-    public JacksonDBObject() {
-    }
+  public JacksonDBObject() {}
 
-    public JacksonDBObject(T object, Class<?> view) {
-        this.object = object;
-        this.view = view;
-    }
+  public JacksonDBObject(T object, Class<?> view) {
+    this.object = object;
+    this.view = view;
+  }
 
-    public T getObject() {
-        return object;
-    }
+  public T getObject() {
+    return object;
+  }
 
-    public void setObject(T object) {
-        this.object = object;
-    }
+  public void setObject(T object) {
+    this.object = object;
+  }
 
-    public Class<?> getView() {
-        return view;
-    }
+  public Class<?> getView() {
+    return view;
+  }
 
-    public void setView(Class<?> view) {
-        this.view = view;
-    }
-
-    @Override
-    public Object get(String key) {
-        if ("_id".equals(key)) {
-            return "Generated _id retrieval not supported when using stream serialization";
```

Comment:
So MongoDB does support generated _id retrieval with 2.6? Maybe we should leave this section of code if you're using MongoDB 2.4?